packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1788,19 +1786,19 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       expect(data.length, equals(4));
       int id = data[0];
       int count = data[1];
-      _CreationLocation location = knownLocations[id];
+      _CreationLocation location = knownLocations[id]!;
       expect(location.file, equals(file));
       // ClockText widget.
-      expect(location.line, equals(55));
+      expect(location.line, equals(53));
```

Comment:
Why did these change?

Reply:
These are referring to code lines within the test file itself and since I removed two lines at the top of the file, the line numbers changed.